wix-media-v2/wix-media-v2/Files/guides/upload-api.md

```diff
@@ -82,6 +82,8 @@ async function uploadMyFile(uploadUrl, fileContent) {
 }
 ```
 
+## Add Uploaded Media
+In order to add the uploaded media files to your Wix site, use [getFileDescriptor()](https://www.wix.com/velo/reference/wix-media-v2/files/getfiledescriptor) to retrieve the compatable media URI. Pass in `data.file.id` from the [response object](#Response) as the only parameter.  
```

Comment:
Code font for function names, always

```suggestion
In order to add the uploaded media files to your Wix site, use [`getFileDescriptor()`](https://www.wix.com/velo/reference/wix-media-v2/files/getfiledescriptor) to retrieve the compatable media URI. Pass in `data.file.id` from the [response object](#Response) as the only parameter.  
```

Reply:
Also don't forget what we said about "data" not being part of the response object